src/os/bluestore/BlueStore.cc

```diff
@@ -2768,7 +2823,29 @@ void BlueStore::Onode::flush()
 }
 
 // =======================================================
-
+// WriteContext
+ 
+/// Checks for writes to the same pextent within a blob
+bool BlueStore::WriteContext::has_conflict(
+  BlobRef b,
+  uint64_t loffs,
+  uint64_t blob_len)
+{
+  loffs = P2ALIGN(loffs, blob_len);
```

Comment:
min_alloc_size is a power of 2, but the blob length needn't be... if min_alloc_size is 64k the blob could be 196k